bin/webpack-configs.js

```diff
@@ -439,6 +348,123 @@ const getPaymentsConfig = ( options = {} ) => {
 						},
 					},
 				},
+				{
+					test: /\.s[c|a]ss$/,
+					use: {
+						loader: 'ignore-loader',
+					},
+				},
+			],
+		},
+		plugins: [
+			new ProgressBarPlugin(
+				getProgressBarPluginConfig(
+					'Payment Method Extensions',
+					options.fileSuffix
+				)
+			),
+			new DependencyExtractionWebpackPlugin( {
+				injectPolyfill: true,
+				requestToExternal,
+				requestToHandle,
+			} ),
+			new NormalModuleReplacementPlugin(
+				/dashicon/,
+				( result ) => ( result.resource = dashIconReplacementModule )
+			),
+		],
+		resolve,
+	};
+};
+
+const getStylingConfig = ( options = {} ) => {
+	let { fileSuffix } = options;
+	const { alias, resolvePlugins = [] } = options;
+	fileSuffix = fileSuffix ? `-${ fileSuffix }` : '';
+	const resolve = alias
+		? {
+				alias,
+				plugins: resolvePlugins,
+		  }
+		: {
+				plugins: resolvePlugins,
+		  };
+	return {
+		entry: getEntryConfig( 'styling', options.exclude || [] ),
+		output: {
+			devtoolNamespace: 'wc',
+			path: path.resolve( __dirname, '../build/' ),
+			filename: `[name]-style${ fileSuffix }.js`,
+			library: [ 'wc', 'blocks', '[name]' ],
+			libraryTarget: 'this',
+			// This fixes an issue with multiple webpack projects using chunking
+			// overwriting each other's chunk loader function.
+			// See https://webpack.js.org/configuration/output/#outputjsonpfunction
+			jsonpFunction: 'webpackWcBlocksJsonp',
+		},
+		optimization: {
+			splitChunks: {
+				minSize: 0,
+				cacheGroups: {
+					editor: {
+						// Capture all `editor` stylesheets and the components stylesheets.
+						test: ( module = {} ) =>
+							module.constructor.name === 'CssModule' &&
+							( findModuleMatch( module, /editor\.scss$/ ) ||
```

Comment:
This is pretty cool, automatically building `editor.css` from `editor.scss` sources. Should we document this somewhere? Not sure if this is new, but either way it's a convention that we must follow when implementing new blocks. (if I understand correctly)

Reply:
It's not really new but I think it was not documented anywhere. Added some docs in #2953.